app/controllers/signatures_reports_controller.rb

```diff
@@ -2,15 +2,29 @@
 
 class SignaturesReportsController < ApplicationController
   before_action :authenticate_user!
+  expose(:work_sites) do
```

Comment:
Manipulating arrays of tuples isn't a common practice in ruby (code tends to be very object oriented and method-focused), so I worry that having this in the main body of the controller will be confusing down the line. I know that the select box takes this format for its input, but can we isolate this so that it doesn't get tangled up in the the rest of the application logic?

Since it's basically a scope on WorkSite, maybe something like this?

```ruby
class WorkSite
  # ...
  def select_box_input
    fallback = ['All Sites', -1] # see justification for -1 in another comment
    addresses = where(active: true).order(:address).pluck(:address, :id) # like your #map, but at the db level
    addresses.unshift fallback
  end
end
```

It's not a perfect solution (there's still some array manipulation going on later in the controller), but this provides a clear place for the tuple-related code to live and discourages mingling.